Symfony/CS/AbstractFunctionReferenceFixer.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS;
+
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @internal base class for function reference fixers
+ *
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ */
+abstract class AbstractFunctionReferenceFixer extends AbstractFixer
+{
+    /**
+     * Looks up Tokens sequence for suitable candidates and delivers boundaries information,
+     * which can be supplied by other methods in this abstract class.
+     *
+     * @param string $functionNameToSearch
+     * @param Tokens $tokens
+     * @param int    $start
+     * @param int    $end
```

Comment:
Was not intended to be optional. Would you advice to make them optional (and take 0, $tokens->count() as defaults)?